tasks/app/controllers/people_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PeopleController < ApplicationController
+  before_filter :last_page
+
+  def index
+    @people = Person.all
+  end
+
+  def show
+    id = params[:id]
+    @person = Person.find(id)
+  end
+
+  def showtasks
+    id = params[:id]
+    @person = Person.find(id)
+    @tasks = Person.find(id).tasks
```

Comment:
For this we could reuse the `@person` variable as it should be the same as calling `Person.find(id)` again.